protocols/earthfund/overview.md

```diff
@@ -0,0 +1,26 @@
+# Earthfund
+
+EarthFund is a DAO-controlled crowdfunding platform for projects and ideas that can build a better tomorrow. It was built to give everybody a meaningful voice in the shape of their future.
+
+We're currently building a fundraising platform on the blockchain. Decentralized and autonomous, but with all the user-friendly features of Web2.0, EarthFund is a platform designed for mass adoption. 
+
+Once live, Earthlings will be able to join communities for causes they’re passionate about, fundraise the money they need, and vote as a collective which world-changing projects get the funding they need to make a difference.
+
+## Governance
+
+Our governance is broken into 4 workstreams and a central group of nominated counselors who oversee the development of the platform. Within a year of launch, the council will be replaced by the community, ensuring EarthFund is fully autonomous.
+
+Voting is handled off-chain. This ensures the results are verifiable and the voting process is tamper-resistant. (Votes and relayer receipts stored in IPFS). Once consensus has been reached, the result is then executed on-chain. 
+
+Proposals are created and then put for a vote. Before that, we allow for discussion and soft consensus.
+
+## More about EarthFund DAO
+
+[Website](https://earthfund.io)
+
+[Twitter](https://twitter.com/EarthFund_io)
+
+[Telegram](https://t.me/earthfund)
+
+[Discord](https://discord.gg/earthfund)
+
```

Comment:
Could you please add the snapshot URL?

Reply:
@Rashmi-278 merging the PR, can you make this change in another PR.

Reply:
@cryptosabago this oeverview is for reference and we post this on arweave so we will have that added there before the release.